vnext/local-cli/src/generator-common/index.ts

```diff
@@ -351,39 +216,39 @@ export function copyAndReplaceWithChangedCallback(
           contentChanged,
         );
```

Comment:
Future cleanup: Now that you have made the prompt case is a lot cleaner, should we merge these tow methods and pass in `alwaysOverwrite`?